src/Alarm.cpp

```diff
@@ -99,7 +99,7 @@ bool Alarm::isTripped(){
 }
 
 void Alarm::trigger(){
-  Logger::Log("Alarm has been triggered!");
+  Serial.write(34005); //34005 = Command, Info, Excuet Command: Trigger
```

Comment:
Two things:
1. You've got a typo in `Excuet`.
2. Are you meaning to send a command to the Pi to have the Pi execute something? I would have thought this would have been just a logging statement instead of sending a command for the Pi to execute. Or if that was the intention then it should _also_ send a logging message.